cmd/install.go

```diff
@@ -18,9 +18,10 @@ type installCmd struct {
 }
 
 type installOpts struct {
-	force    bool
-	provider string
-	all      bool
+	all       bool
+	force     bool
+	provider  string
+	latestURL string
```

Comment:
we should move this `latestURL` inside the generic provider only since it doesn't apply to all of them